apps/pwabuilder/src/script/components/publish-pane.ts

```diff
@@ -75,9 +75,9 @@ export class PublishPane extends LitElement {
       title: 'Windows',
       factoids: [
```

Comment:
I feel like factoids, or maybe even this entire array should be moved to a different file from the component? For complexity and making it easier to read, I would try splitting out this logic into a service `whatevs.ts` that the component then imports and uses.

Reply:
I'll open an issue for this to be taken care of in the future.